src/Components/MessageList.jsx

```diff
@@ -38,15 +33,15 @@ class MessageList extends React.Component {
 
     return (
       <List celled>
-        {messageIds.map((msgId) => {
-          const { avatarImage, name, text, timestamp } = messages[msgId];
+        {messageIds.map(msgId => {
+          const { userMapId, message, messageTimestamp, rawTs } = messages[msgId];
           return (
             <Message
-              key={msgId}
+              key={rawTs}
```

Comment:
this is a momentary solution since there isn't a unique id. Please document in the code as a TODO, as well as create a trello task for it. thanks.